website/versioned_docs/version-0.62/testing.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
this file should be deleted and links to it replaced with links to the repo version

Reply:
Removed, linking to it or similar will be tracked with #149